python-package/lightgbm/engine.py

```diff
@@ -421,8 +424,8 @@ def cv(params, train_set, num_boost_round=100,
        folds=None, nfold=5, stratified=True, shuffle=True,
        metrics=None, fobj=None, feval=None, init_model=None,
        feature_name='auto', categorical_feature='auto',
-       early_stopping_rounds=None, fpreproc=None,
-       verbose_eval=None, show_stdv=True, seed=0,
+       early_stopping_rounds=None, early_stopping_threshold=0.0,
```

Comment:
I guess this PR was developed before #4574 where it is proposed to eliminate advanced arguments of high-level functions and encourage users to use callbacks directly. If that proposal will be accepted, I think `early_stopping_threshold` shouldn't be exposed in training API.

Reply:
Yes, I started working on this earlier but I agree with #4574 and I think it will also be easier to test (just testing the callback on one interface instead of testing the argument in `train`, `cv` and the scikit-learn estimators) and to document. I'll update this PR accordingly.

Reply:
oh good point, I didn't make that connection about #4574. Thanks @StrikerRUS